crazyflie/config/crazyflies.yaml

```diff
@@ -1,14 +1,64 @@
-cf231:
-  initial_position:
-  - 0
-  - 0
-  - 0
-  type: cf21
-  uri: radio://0/80/2M/E7E7E7E7E7
-cf5:
-  initial_position:
-  - 0
-  - -0.5
-  - 0
-  type: cf21
-  uri: radio://0/80/2M/E7E7E7E705
+# named list of all robots
+robots:
+  cf231:
+    enabled: true
+    uri: radio://0/80/2M/E7E7E7E7E7
+    initial_position: [0, 0, 0]
+    type: cf21
+    # firmware_params:
+    #   kalman:
+    #     pNAcc_xy: 1.0 # default 0.5
+  cf5:
+    enabled: false
+    uri: radio://0/80/2M/E7E7E7E705
+    initial_position: [0, -0.5, 0]
+    type: cf21
+    # firmware_params:
+    #   kalman:
+    #     pNAcc_xy: 1.0 # default 0.5
+
+# Definition of the various robot types
+robot_types:
+  cf21:
+    motion_capture:
+      enabled: true
+      # only if enabled; see motion_capture.yaml
+      marker: default_single_marker
+      dynamics: default
+    big_quad: false
+    battery:
+      voltage_warning: 3.8  # V
+      voltage_critical: 3.7 # V
+
+  cf21_mocap_deck:
+    motion_capture:
+      enabled: true
+      # only if enabled; see motion_capture.yaml
+      marker: mocap_deck
+      dynamics: default
+    big_quad: false
+    battery:
+      voltage_warning: 3.8  # V
+      voltage_critical: 3.7 # V
+
+# global settings
+settings:
```

Comment:
could be more descriptive, such as 'server_settings' ?

Reply:
What do you think about "all" or "all_robots"? to make the hierarchy clearer?

Reply:
Yeah that is good too! I actually thought of that too when developing the cflib server, to name it "all/firmware-params" so the logic would be simpler. So that already indicates that that is the natural name.

For a little more descriptivity, all_robots is better, but /all is more awesome 😁

Reply:
Ah we could also do this for the swarm services like /all/takeoff, but for another PR then 😊